src/runtime/virtcontainers/pkg/compatoci/utils_test.go

```diff
@@ -145,3 +179,55 @@ func TestGetConfigPath(t *testing.T) {
 	configPath := getConfigPath(tempBundlePath)
 	assert.Equal(t, configPath, expected)
 }
+
+func TestParseConfigHSON(t *testing.T) {
```

Comment:
nit: Seems like a typo here. Should be `TestParseConfigJSON` here.

Reply:
Thanks, I will fix it